modules/integrate-splunk-configure-splunk.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-splunk.adoc
+:_module-type: PROCEDURE
+[id="integrate-splunk-configure-splunk_{context}"]
+= Configuring Splunk
```

Comment:
I think it would be helpful to be more specific with what you're doing in this procedure